### ISSUE TITLE
Not enough space on the disk.

### ISSUE THREAD

**Issue by PsuperGirl** [Nesvilab/FragPipe#1529]
- Describe the issue or question:
Hi FragPipe team,

Recently, I am trying to search for some DDA data on phosphorylation sites. However, it also complains that there is not enough space on the disk or insufficient memory, although there was at least 700GB on my drive.
I have noticed that someone had similar issues with search phospho-data 4 years ago. I am wondering how it was fixed. It would be great if you can give some suggestions.

I have attached two log files here for both errors.
[log_2024-04-08_18-36-52.txt](https://github.com/Nesvilab/FragPipe/files/14930031/log_2024-04-08_18-36-52.txt)
[log_2024-04-10_10-32-56.txt](https://github.com/Nesvilab/FragPipe/files/14930032/log_2024-04-10_10-32-56.txt)

Thank you very much in advance!

Best,
Juanjuan

Juanjuan Wang, PhD
Postdoc
juanjuan.wang@cpr.ku.dk

University of Copenhagen
Novo Nordisk Foundation Center for Protein Research
Proteomics Program
Blegdamsvej 3B,
2200 Copenhagen, Denmark

**Comment by amingson** [comment]
感觉像是线程数太高了，我平常都用20线程

**Comment by PsuperGirl** [comment]
> 感觉像是线程数太高了，我平常都用20线程

线程是什么？这个在哪里可以设置呢？

**Comment by amingson** [comment]
> > 感觉像是线程数太高了，我平常都用20线程
> > Do you mean this parameter？
> > workflow.ram=0
> > workflow.threads=63 change to 20

Yes !  And limit ram below the computer's max.  I believe this would be useful!

Best,
Anler

**Comment by PsuperGirl** [comment]
> > > 感觉像是线程数太高了，我平常都用20线程
> > > Do you mean this parameter？
> > > workflow.ram=0
> > > workflow.threads=63 change to 20
> 
> Yes ! And limit ram below the computer's max. I believe this would be useful!
> 
> Best, Anler

谢谢！

**Comment by fcyu** [comment]
> However, it also complains that there is not enough space on the disk or insufficient memory, although there was at least 700GB on my drive.

Your 700 GB is the hard drive space, not memory. FragPipe detected 79 GB free memory according to your log file.

`workflow.ram=0` should be OK most of time because FragPipe detects the amount of free memory automatically, which is 79 GB in your case. 

Best,

Fengchao
